Gemfile.lock

```diff
@@ -64,6 +64,32 @@ GEM
     dotenv-rails (2.1.1)
       dotenv (= 2.1.1)
       railties (>= 4.0, < 5.1)
+    dry-configurable (0.1.6)
```

Comment:
This is a lot of dependencies for just one gem. If you're gonna use more of these `dry-` gems in the future its cool but if its only gonna be used in a few validations then ActiveModel::Validations is probably good enough